src/js/config/config.js

```diff
@@ -29,4 +30,15 @@ if (process.env.GULP_ENV === "development") {
       "to enable a development configuration.");
   }
 }
+
+// Convert relative URLs to absolute URLs for node-fetch
+if (global.document != null) {
```

Comment:
Oh maybe I got it wrong then. Isn't this only needed by `node` (ie for our tests?)